packages/experiments/src/components/PersonaCoin/PersonaCoin.styles.ts

```diff
@@ -0,0 +1,39 @@
+import { IPersonaCoinComponent } from './PersonaCoin.types';
+// tslint:disable-next-line
+import { initialsColorPropToColorCode } from '../../../../../packages/office-ui-fabric-react/src/components/Persona/PersonaInitialsColor';
```

Comment:
We need to be careful of this sort of import path (I think). Make sure to reference the package prior to check-in.

Reply:
Good catch, thanks Kevin :D